build.sh

```diff
@@ -77,7 +77,8 @@ usage() {
 	exit 1
 }
 
-export BUILDER_ROOT=$(realpath $(dirname ${0}))
+export BUILDER_ROOT
+BUILDER_ROOT=$(realpath "$(dirname "${0}")")
```

Comment:
what is the reason to split it in 2 lines? Also, I cannot think about a reason to have empty $0, then these extra quotes are unnecessary IMO